components/nifosg/controller.hpp

```diff
@@ -283,9 +283,17 @@ namespace NifOsg
     {
     private:
         Vec3Interpolator mData;
+        enum TargetColor
+        {
+            Ambient,
+            Diffuse,
+            Specular,
+            Emissive
```

Comment:
Assign specific values to enum elements. Now it seems like the order is not important which actually is not a case because `(matctrl->flags >> 4) & 7` gives specific values.

Reply:
They're implicitly initialized to 0, 1, 2, 3 in that order though, and it is a standard C++ syntax, I'm not sure how would it seem the order is not important.

Reply:
It is a standard syntax, but if order doesn't matter same approach could be used. And this might be confusing.